src/pages/examSetter/AddCandidate.tsx

```
@@ -1,10 +1,297 @@
+      </Row>
+
+      <Divider />
+
+      <Row>
+        <Col span={24}>
+          <List
+            header={<strong>Selected Candidates</strong>}
+            bordered
+            dataSource={selectedCandidates}
+            renderItem={(item) => (
+              <List.Item actions={[<a onClick={() => removeCandidate(item.id)}>Remove</a>]}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)